lib/CallBackery/qooxdoo/callbackery/source/class/callbackery/ui/plugin/CardList.js

```diff
@@ -111,7 +116,7 @@ qx.Class.define("callbackery.ui.plugin.CardList", {
                 }
                 busy.vanish();
                 that._loading--;
-            }, 'getPluginData', this._cfg.name, 'allCardData', currentFormData);
+            }, 'getPluginData', this._cfg.name, 'allCardData', parentFormData, { currentFormData: this._form.getData()});
```

Comment:
ich haette jetzt HIER einen neuen call gemacht, da du ja die api aenderst und dann im abstract backend einen fall back implementiert in der getAcclCardData methoded der abstrct class die dann die alte implementation aufzurufen versucht ...